### PR TITLE
Fix addTimeCommand bug

### PR DESCRIPTION
**Pull request by larainezo**

### CODE REVIEW

**Review by nyankoclaws — APPROVED** [review]
LGTM

File: src/test/java/seedu/address/logic/commands/AddTimeCommandTest.java [hunk 1]
```diff
@@ -78,15 +79,13 @@ public void execute_emptyFreeTime_failure() {
...
-        assertTrue(outOfBoundIndex.getZeroBased() < model.getAddressBook().getPersonList().size());
-
-        AddTimeCommand addTimeCommand = new AddTimeCommand(outOfBoundIndex,
-                new AddPersonFreeTimeDescriptorBuilder().withFreeTimeTags(VALID_FREE_TIME_TAG_BOB).build());
+        assertTrue(INDEX_FIRST_PERSON.getZeroBased() < model.getAddressBook().getPersonList().size());
 
+        AddTimeCommand addTimeCommand = new AddTimeCommand(INDEX_FIRST_PERSON,
+                new AddPersonFreeTimeDescriptorBuilder().withFreeTimeTags(VALID_FREE_TIME_TAG_AMY).build());
 
         Person editedPerson = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
```
Comment (PallonCX) [inline src/test/java/seedu/address/logic/commands/AddTimeCommandTest.java:90]: May I know how this test function works?

**Review by donwong2308 — APPROVED** [review]
LGTM

**Review by PallonCX — APPROVED** [review]
LGTM

**Review by JustWeiHao — APPROVED** [review]
LGTM

**Pull request merged**